conf/standalone.conf

```diff
@@ -29,6 +29,8 @@ metadataStoreUrl=
 # Configuration file path for metadata store. It's supported by RocksdbMetadataStore and EtcdMetadataStore for now
 metadataStoreConfigPath=
 
+# Whether we should enable fsync for local metadata store. It's supported by RocksdbMetadataStore for now
```

Comment:
It would be more clear to add this: "If this flag is true, metadata writes will be slower. If this flag is false, and the machine crashes, some recent metadata writes may be lost. Note that if it is just the process that crashes (i.e., the machine does not reboot), no writes will be lost even if it is false."